src/draco/psy/psy_draco_decoder.cpp

```diff
@@ -228,15 +249,13 @@ class MeshDecompression::Impl
     }
 
     Header mDecompressedHeader;
+    uint32_t mCurrIFrameIndex;
     std::shared_ptr<::draco::Mesh> mpMesh;
     std::shared_ptr<::draco::DecoderBuffer> mpBuffer;
     std::unique_ptr<MeshEdgeBreakerDecompression> mpMeshDecompression;
     ::draco::Status mStatus;
 };
 
-MeshDecompression::MeshDecompression(const MeshDecompression&) : mpImpl(nullptr) {};
-MeshDecompression& MeshDecompression::operator=(const MeshDecompression&) { return *this; };
-
```

Comment:
It feels like @nbcong wants to disable the copy? These two functions are essentially doing nothing.
I guess you could make `MeshDecompression::MeshDecompression(const MeshDecompression&) = delete` to see if anything fails compiling.

Reply:
I think putting them in private without an implementation will do the trick, I tested that everything compiles with this code and works

Reply:
Ahh I see they're private already. I thought this was the header file. 👍

Reply:
Sorry, I still not understand what the problems if we make them private and have implementations in .cpp file?
@zy53012127 yes, I want to disable the copy!

Reply:
we don't need an implementation to just disable copy